Sources/PackageModel/Sources.swift

```diff
@@ -51,6 +51,9 @@ public enum SupportedLanguageExtension: String {
     case cc
     case cpp
     case cxx
+    /// Assembly
+    case s
+    case S
```

Comment:
We should also add .asm here, it is a supported extension for assembly in Xcode.

Reply:
I don’t think we have to follow precedent with Xcode or Clang. We already break with Clang and I think that is good for consistency. I haven’t found “asm” to be widespread, but also haven’t don’t an audit. I’d prefer we start conservative, or do an audit to decide.

Reply:
@neonichu / @ddunbar / @jakepetroules what's the verdict here? I'd be in favour if just leaving .s and .S (and no .asm) but I really don't mind

Reply:
We can add .asm later if needed. A restrictive start is better than an open-ended one anyway.

Reply:
I prefer .s and .S only